watson.completion

```diff
@@ -76,6 +76,18 @@ _watson_complete () {
             COMPREPLY=($(compgen -W "$tags" -- ${cur}))
           fi
           ;;
+        rename)
+          case "$3" in
+            project)
+              projects="$(watson projects | sed -e 's/ /\\\\ /g')"
+              COMPREPLY=($(compgen -W "$projects" -- ${cur}))
+              ;;
+            tag)
+              tags="$(watson tags | sed -e 's/ /\\\\ /g')"
+              COMPREPLY=($(compgen -W "$tags" -- ${cur}))
+              ;;
+          esac
+          ;;
       esac
       ;;
   esac
```

Comment:
`rename` needs to be added to the list of commands in line 9 of this file as well.